main.go

```diff
@@ -20,12 +20,11 @@ func main() {
 
 	masTree := tree.NewTree(40, company)
 
-	if live {
-		for true {
-			masTree.Print()
-			time.Sleep(2 * time.Second)
-		}
-	} else {
+	for true {
```

Comment:
8 -> 6 lines. removed else.

Reply:
Look good, but I really don't like do check in every loop.